examples/configs/config-dev.json

```diff
@@ -4,9 +4,10 @@
 		"zookeeper"
 	],
 	"clusterId": "iudx-rs-proxy-cluster",
-	"host": "",
 	"commonConfig" :  {
-           "dxApiBasePath" : "/ngsi-ld/v1"
+         "dxApiBasePath" : "/ngsi-ld/v1",
+         "dxCatalogueBasePath": "/iudx/cat/v1",
+         "dxAuthBasePath": "/auth/v1"
```

Comment:
We cannot delete these 2 properties they are used.
`dxCatalogueBasePath` [here](https://github.com/datakaveri/iudx-rs-proxy/blob/main/src/main/java/iudx/rs/proxy/authenticator/JwtAuthenticationServiceImpl.java#L68), and
`dxAuthBasePath` [here](https://github.com/datakaveri/iudx-rs-proxy/blob/main/src/main/java/iudx/rs/proxy/authenticator/AuthenticationVerticle.java#L121)

Reply:
> We cannot delete these 2 properties they are used. `dxCatalogueBasePath` [here](https://github.com/datakaveri/iudx-rs-proxy/blob/main/src/main/java/iudx/rs/proxy/authenticator/JwtAuthenticationServiceImpl.java#L68), and `dxAuthBasePath` [here](https://github.com/datakaveri/iudx-rs-proxy/blob/main/src/main/java/iudx/rs/proxy/authenticator/AuthenticationVerticle.java#L121)

I have added those properties  as it was not there in config and deleted the "host" config option